redbox-core/tests/graph/test_state.py

```diff
@@ -402,7 +410,7 @@ def test_tool_calls_reducer(a: ToolState, b: ToolState, expected: ToolState):
         ),
     ],
 )
-def test_merge_redbox_state_updates(a: RedboxState, b: RedboxState, expected: RedboxState):
+def test_merge_redbox_state_updates(a: dict, b: dict, expected: dict):
```

Comment:
This test never used the documents reducer for some reason, now it does

Reply:
much clearer :+1: